packages/adaptive_scaffold/lib/src/adaptive_scaffold.dart

```diff
@@ -309,15 +308,19 @@ class AdaptiveScaffold extends StatefulWidget {
       int currentIndex = 0,
       double iconSize = 24}) {
     return Builder(
-        builder: (_) => BottomNavigationBar(
-            currentIndex: currentIndex,
-            iconSize: iconSize,
-            items: destinations
-                .map((NavigationDestination e) => _toBottomNavItem(e))
-                .toList()));
+      builder: (_) {
```

Comment:
I wonder if it is necessary to use a builder here if we don't use its BuildContext

Reply:
Let's discuss this, but address it in another PR, I just want to get these clean-ups in.